src/main/java/de/tum/www1/orion/vcs/OrionGitAdapter.kt

```diff
@@ -258,11 +258,11 @@ object OrionGitAdapter {
 
     private fun resetAndPull(module: Module) {
         ProgressManager.getInstance().run(object :
-            Task.Modal(module.project, "Updating your exercise files...", false) {
+            Task.Modal(module.project, "Updating Your Exercise Files...", false) {
```

Comment:
I am not a big fan of capital case in normal sentences, I would leave it as it was

Reply:
Fixed! :)